cfpb/consumerfinance.gov
# [Paying for College] Dynamic Disclosures Functional Tests

This PR adds the first bevy of cypress tests for the Dynamic Disclosures portion of Paying for College.

## Additions
- Adds tests in `test/cypress/integration/paying-for-college/disclosures`
- Adds new fixtures in `test/cypress/fixtures/paying-for-college`

## How to test this PR
1. Set this branch up locally
2. Try `yarn run cypress open`
3. Check out those disclosures tests!

## Notes and todos
- There could be more tests here, and I have some notes for possible additional tests I might add

## Checklist
